fpga_interchange/examples/devices/xc7a35t/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+generate_chipdb(part xc7a35tcsg324-1)
```

Comment:
somewhat incidental to this PR, but given the routing graph is the same, having one chipdb across all packages would make more sense imo

Reply:
Definetely makes sense, I am not sure whether the part-specific data is required somewhere, but for now I changed it such that only one device is generated (for xc7a35t and xc7a50t, the common device is the 50t)

Reply:
So because of how the device database is structured, the current code in python-fpga-interchange will generate a chipdb for all packages present in the input device database.  This does mean that if you request a xc7a35tcsg324-1, you get a database for all `xc7a35t` devices.